app/api/ms_api_utils.py

```diff
@@ -1,12 +1,9 @@
 import logging
-import requests
 from http import HTTPStatus
+import requests
+from flask import json
```

Comment:
import order seems incorrect, should be from http, then from flask then requests. Please check this

Reply:
Checked with pylon run.py, but both orders seems to be accepted just fine.
http => requests => flask
<img width="794" alt="Screen Shot 2020-06-22 at 12 41 59 am" src="https://user-images.githubusercontent.com/29667122/85227549-8b2b1b80-b421-11ea-9d6d-36d4991a66d5.png">

http => flask => requests
<img width="778" alt="Screen Shot 2020-06-22 at 12 42 54 am" src="https://user-images.githubusercontent.com/29667122/85227564-9da55500-b421-11ea-8d3d-c52575672da4.png">

Reply:
Awesome, great!